_posts/2019-11-13-cockpit-207.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 207
+author: kkoukiou
+date: 2019-10-30
+tags: cockpit linux
+slug: cockpit-207
+category: release
+summary: Cockpit with authentication and storage page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Cockpit's web server now accepts EC certificates
+
+To specify the [Elliptic Curve Cryptography](https://wiki.openssl.org/index.php/Elliptic_Curve_Cryptography) certificate
+the web service should use, simply drop a file with the extension .cert in the `/etc/cockpit/ws-certs.d` directory.
+
+### Storage: All Software Devices will be listed now in a single panel
```

Comment:
"List all software devices in a single panel"

Reply:
I guess with the prefix, like:

"Storage: List all software devices in a single panel"

...right?

Reply:
Perhaps add a rationale with a condensed variant of the https://github.com/cockpit-project/cockpit/pull/11809 description? Maybe "This avoids showing empty panels for types of devices that you don't have".